platform/podjs/src/manifest-parser/src/manifest.ts

```diff
@@ -67,22 +57,24 @@ const featureDecoder = Decode.type({
     translations: Decode.UnknownRecord,
 });
 
-export async function readManifest(packageManifest: Record<string, unknown>): Promise<Manifest> {
-    // There is no 'version' property in real feature manifests at this time, but to make this code
-    // happy, we're simply adding one if it's missing.
-    const manifestWithVersion = { ...packageManifest };
-    if (!("version" in manifestWithVersion)) manifestWithVersion.version = "1.0.0";
+export async function readManifest(
+    packageManifest: Record<string, unknown>
+): Promise<Manifest> {
+    const manifest = { ...packageManifest };
```

Comment:
Not entirely clear what this achieves... `expect` can take any kind of `I` as an argument, and it's directly passed to the decoder.